src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function containerSize($card-width, $card-magrin) {
+  @return $card-width * 4 + $card-magrin * 3;
```

Comment:
to express these values with variables is better: 4 (let's call it `x`) — number of cards per row, 3 — number of gaps or (`x` - 1)

Reply:
Fixed